src/commcare_cloud/ansible/roles/kafka/tasks/main.yml

```diff
@@ -61,27 +61,8 @@
 - name: Add Kafka binaries to PATH
   copy: src=kafka.sh dest=/etc/profile.d/ owner=root group=root mode=644
 
-- name: Add ruby ppa
-  apt_repository: repo='ppa:brightbox/ruby-ng'
-  register: add_ruby_ppa
-  tags:
-    - kafkat
-
-- name: Update package list for ruby
-  apt: update_cache=yes
-  when: add_ruby_ppa|changed
-  tags:
-    - kafkat
-
-- name: Install ruby
-  apt: name="{{ item }}" state=present
-  become: yes
-  with_items:
-    - ruby2.3
-    - ruby2.3-dev
-  tags:
-    - kafkat
-  ignore_errors: '{{ ansible_check_mode }}'
+- include: roles/common/tasks/install_ruby.yml
```

Comment:
This is great but rather than using an include you can make the role depend on the ruby role by adding a dependency to `meta/main.yml`

Reply:
I thought this was too small to make it into a role. (This would require creating a ruby role, no?). Is there any issue using this style of `include` ?

Reply:
Here's a similar role for git: https://github.com/dimagi/commcare-cloud/blob/810e71308d7eec86a0f5859d416e1b80c717f7ab/src/commcare_cloud/ansible/roles/git/tasks/main.yml

The reason to make it a role is that it follows the ansible pattern for reuse better.

Reply:
Updated.